doc/man7/OSSL_PROVIDER-FIPS.pod

```diff
@@ -333,7 +333,7 @@ The FIPS module passes the following descriptions(s) to OSSL_SELF_TEST_onbegin()
 
 "Module_Integrity" and "Install_Integrity" use this.
 
-=item "RSA" (B<OSSL_SELF_TEST_DESC_PCT_RSA_PKCS1>)
```

Comment:
You have to leave the old entry alone and just add the new one since this is a public header.